tests/app/forms/test_questionnaire_form.py

```diff
@@ -14,7 +16,8 @@ class TestQuestionnaireForm(AppContextTestCase):  # noqa: C901  pylint: disable=
 
     @staticmethod
     def _error_exists(answer_id, msg, mapped_errors):
-        return any(a_id == answer_id and msg in ordered_errors for a_id, ordered_errors in mapped_errors)
+        msg = str(msg) if isinstance(msg, LazyString) else msg
```

Comment:
No need to check `isinstance(msg, LazyString)` here is there? Just always run `str(msg)`. It'll just return immediately if `msg` is already a string.

Reply:
I wanted to be a bit more explicit as to why i'm converting it but maybe overkill.